src/core/src/package_managers/YumPackageManager.py

```diff
@@ -142,6 +142,9 @@ def get_all_updates(self, cached=False):
 
     def get_security_updates(self):
         """Get missing security updates"""
+        if self.__is_image_rhel8_higher_skip_security_plugin():
+            return [], []
+
         self.composite_logger.log("\nDiscovering 'security' packages...")
         self.install_yum_security_prerequisite()
```

Comment:
The purpose of the fix is to not run Line 149 (and only Line 149) " self.install_yum_security_prerequisite() " if the RHEL version is 8+.

What the code here does is it skips everything in this function which is incorrect.

So. ---
if not rhel8+:
    self.install...

continue after that regardless

Reply:
logic is apply on self.install_yum_security_prerequisite